exasol_transformers_extension/udfs/model_downloader_udf.py

```diff
@@ -1,8 +1,40 @@
+import os
+import tempfile
+import transformers
+from pathlib import PurePosixPath
+from exasol_bucketfs_utils_python.bucketfs_factory import BucketFSFactory
 
 
 class ModelDownloader:
-    def __init__(self, exa):
+    def __init__(self, exa, downloader_method=transformers.AutoModel):
```

Comment:
does it download different models, when you are using AutoModel and AutoTokenizer

Reply:
AutoModel and AutoTokenizer seems different models. The AutoTokenizer model should be additionally saved   for each model.